NuGet.config

```diff
@@ -4,7 +4,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
-    <!--  End: Package sources from dotnet-aspnetcore -->
```

Comment:
Do not remove this line. It should appear after the line you added.

Reply:
Just reverted